### PR TITLE
Added config for Raspberry Pi from another RT882XX Makefile

### PR DESCRIPTION
**Pull request by sigkill**
From another Makefile I compared, and have just tested building this driver on vanilla raspbian.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -77,7 +77,8 @@ CONFIG_RTW_SDIO_PM_KEEP_POWER = y
 ###################### MP HW TX MODE FOR VHT #######################
 CONFIG_MP_VHT_HW_TX_MODE = n
 ###################### Platform Related #######################
-CONFIG_PLATFORM_I386_PC = y
+CONFIG_PLATFORM_I386_PC = n
+CONFIG_PLATFORM_ARM_RPI = y
```
Comment (tpircher-zz) [inline Makefile:81]: The change is good, thanks. However changing the PLATFORM setting will break the build for everyone else. Can the change at lines 80/81 be modified to keep "CONFIG_PLATFORM_I386_PC = y" and to set "CONFIG_PLATFORM_ARM_RPI = n"?

File: Makefile [hunk 2]
```diff
@@ -898,6 +899,17 @@ ifeq ($(CONFIG_WIFI_MONITOR), y)
...
 endif
 
+ifeq ($(CONFIG_PLATFORM_ARM_RPI), y)
+    EXTRA_CFLAGS += -DCONFIG_LITTLE_ENDIAN
+    EXTRA_CFLAGS += -DCONFIG_IOCTL_CFG80211 -DRTW_USE_CFG80211_STA_EVENT
+    ARCH ?= arm
+    CROSS_COMPILE ?=
+    KVER ?= $(shell uname -r)
+KSRC := /lib/modules/$(KVER)/build
+MODDESTDIR := /lib/modules/$(KVER)/kernel/drivers/net/wireless/
```
Comment (tpircher-zz) [inline Makefile:909]: I believe these settings should be provided by the user, and should not be preset by the Makefile

**Comment by tpircher-zz** [comment]
Thanks for the change! I have added a few commends on the pull request. The main issue that prevents me from merging the change is that it changes the pltform in the default seeting. This will break the build for everyine else. I'd rather see the new ARM setting added, but disabled by default. This can then be enabled when needed.

File: Makefile (hunk 1 above)
Comment (sigkill) [reply Makefile:81]: Sure great suggestion!

Comment (tpircher-zz) [reply Makefile:81]: Hi, do you mind updating your merge request to that effect?

**Comment by sigkill** [comment]
Please do!

**Comment by tpircher-zz** [comment]
I meant can you update your pull request to that effect please?

**Pull request closed without merging**